docs/data.rst

```diff
@@ -118,20 +134,82 @@ In an Output, sequences are automatically resized:
 
 .. testcode::
 
-    output.instance.set_number("my_int_sequence[1]", 10)
+    output.instance.set_number("my_int_sequence[5]", 10) # length is now 5
+    output.instance.set_number("my_int_sequence[4]", 9) # length still 5
 
-For a non-primitive sequence:
+You can clear a sequence:
 
 .. testcode::
 
-    output.instance.set_number("my_sequence[1].x", 10)
+    output.instance.clear_member("my_int_sequence") # my_int_sequence is now empty
 
-To get the length of a sequence (Input only):
+To get the length of a sequence in an Input sample:
 
 .. testcode::
 
     length = input[0].get_number("my_sequence#")
 
+Accessing optional members
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Optional members may have a value or not. They are accessed the same way as
+non-optional members, except that ``None`` is a possible value.
+
+On an Input, any of the getters may return ``None`` if the field is optional:
+
+.. testcode::
+
+    if input[0].get_number("my_optional_long") == None:
```

Comment:
I think (may somebdy else can confirm) that `is` is the preferred way to check for None in python (since None is a singleton.

Reply:
Note: If accepted, this comment would apply to everywhere that `== None:` is used

Reply:
Thanks. You're right. I will change it.